exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -38,13 +38,23 @@ import (
 type prwTelemetry interface {
 	recordTranslationFailure(ctx context.Context)
 	recordTranslatedTimeSeries(ctx context.Context, numTS int)
+	recordRemoteWriteSentBatch(ctx context.Context)
+	setNumberConsumer(ctx context.Context, n int64)
 }
 
 type prwTelemetryOtel struct {
 	telemetryBuilder *metadata.TelemetryBuilder
 	otelAttrs        []attribute.KeyValue
 }
 
+func (p *prwTelemetryOtel) setNumberConsumer(ctx context.Context, n int64) {
+	p.telemetryBuilder.ExporterPrometheusremotewriteConsumers.Record(ctx, n, metric.WithAttributes(p.otelAttrs...))
+}
+
+func (p *prwTelemetryOtel) recordRemoteWriteSentBatch(ctx context.Context) {
+	p.telemetryBuilder.ExporterPrometheusremotewriteSentBatchCount.Add(ctx, 1, metric.WithAttributes(p.otelAttrs...))
```

Comment:
```suggestion
	p.telemetryBuilder. ExporterPrometheusremotewriteSentBatch.Add(ctx, 1, metric.WithAttributes(p.otelAttrs...))
```

That's the name you've added in `telemetryBuilder` 😛

Reply:
Done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37912/commits/f535f8e580206b4f7bf622abd8311b8d09e7716b